walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -112,7 +112,7 @@
         allAttendees.addAll(optionalAttendees);
         
         long duration = request.getDuration();
-        if (duration < 0 || duration >= TimeRange.END_OF_DAY) {
+        if (duration < 0 || duration > TimeRange.END_OF_DAY) {
```

Comment:
Could you also add a test case where duration is whole day?

Reply:
Done! Also added test case for negative duration as well to test same line.